clicache/src/templates/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ cmake_minimum_required(VERSION 3.4)
 project(nativeclient.clicache.templates)
 
 # Set the .NET Target Framework (Note: This should match the build for Apache.Geode.)
-set (DOTNET_TARGET_FRAMEWORK_VERSION "v4.5.2")
+set (DOTNET_TARGET_FRAMEWORK_VERSION ${DOTNET_TARGET_FRAMEWORK_VERSION})
```

Comment:
What is the point of redefining it here to the same value it is already set to?

Reply:
Good catch.